spec/tasks/translation_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+require_relative '../../spec/support/tasks'  
+
+describe 'rake tasks' do
+  before do
+    fake_rails = double()
+    fake_rails.stub(:root) { Pathname.new('spec') }
+    stub_const("Rails", fake_rails)
+  end
+
+  describe 'translation:add_missing', type: :task do
+    let(:task) { Rake::Task["translation:add_missing"] }
+  
+    it 'is executed' do
+      expect { task.execute }.to output.to_stdout
+    end
+
+    it 'triggers i18n task' do
+      allow(I18n::Tasks::CLI).to receive(:start)
+      task.execute
+      expect(I18n::Tasks::CLI).to have_received(:start)
+    end
+  end
+
+  describe 'translation:normalize', type: :task do
+    let(:task) { Rake::Task["translation:normalize"] }
+  
+    it 'is executed' do
+      expect { task.execute }.to_not output.to_stdout
```

Comment:
This surprised me - do we not expect to see any output from this command?
Maybe it's worth adding a `puts "Done!"` at the end of the Rake task, so that we can check that the output contains `Done!`?

Reply:
@ChrisBAshton For `normalize` task this is expected behaviour - task doesn’t return any output, in opposite to `add_missing`. I have been additionally testing this by running tasks on `collections` app.

Reply:
Fair enough 👍